src/markdown-pages/contribute-to-quickstarts/instrument-library/send-events.mdx

```diff
@@ -0,0 +1,170 @@
+---
+title: 'Send Events from Your Library'
+template: 'GuideTemplate'
+description: 'With our Events API you can ingest your data as the first step towards your quickstart contribution.'
+tags:
+  - quickstart
+---
+
+<Intro>
+
+This guide will walk you through how to instrument your application to send event data into the New Relic's [Event API](https://docs.newrelic.com/docs/data-apis/ingest-apis/introduction-event-api/). The Event API allow you to send events to New Relic without a need for an agent. First, let's cover what events are. An event is a record of something happening somewhere. What triggers the event and what data is captured is up to you to customize, which we'll discuss next. 
```

Comment:
> An event is a record of something happening somewhere.

This is pretty vague. Could you provide an example?

Reply:
I added some better context here, so I'll resolve this.